app/assets/stylesheets/hybox.scss

```diff
@@ -1,26 +1,141 @@
-$brand-primary: #629dd1 !default;
-$jumbotron-heading-font-size: 2.5em;
-$jumbotron-heading-color: #fff;
-$jumbotron-color: #d8d8d8;
 
-.jumbotron {
-  background-image: url("http://hydrainabox.projecthydra.org/images/dark_tint.png"), url("http://hydrainabox.projecthydra.org/images/fractal.jpg");
+#masthead,
+#footer {
+  background-color: #3c3c3c;
+  border: 0;
+
+  a,
+  p {
+    color: #ccc;
+    font-weight: 500;
+  }
+}
+
+#masthead {
+  #logo {
```

Comment:
This styling makes me think the upstream masthead might be using the wrong markup.